BusinessLayer/Interfaces/INewsService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BusinessLayer.Models;
+using DataLayer.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BusinessLayer.Interfaces
+{
+    public interface INewsService
```

Comment:
Please, when you create service think about responsibility of service
Currently you can call it like IManageNewService or something like that